templates/terraform/command-center-project/service-accounts.tf

```
@@ -0,0 +1,11 @@
+module command_center_service_account {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google.target
+  }
+
+  account_id = var.service_account_id
+  display_name = "Service account to run GKE system pods"
+  vault_path = "${var.vault_prefix}/gcs/sa-key"
```

I wouldn't include `gcs` in this path, since the account doesn't have permissions on those resources. I'd also be more specific with the "leaf" since there are going to be multiple accounts. My suggestion:
```suggestion
  vault_path = "${var.vault_prefix}/service-accounts/gke-runner"
```